### DISCUSSION TITLE
Image Resize

### DISCUSSION THREAD

**Discussion by WenLonG12345** [keystonejs/keystone#7749] [Feature requests]
Hi, I found this library very nice and beautiful, thanks for all the hard work!
However, I found there is no image resizing function on `image()`. Is there a function for this that I missed out from the documentation? or this is a feature request? 
Do let me know more and hope to see more in this CMS!

**Comment by AliceRossa** [comment]
Is there anything new about this feature?
How can we resize an oversized image when uploading?

**Comment by AliceRossa** [comment]
Are there any updates on this?
I'm also interested in understanding how to resize and optimize uploaded images.

**Comment by gautamsi** [comment]
you want this which uploads 4 different size of the images
https://www.npmjs.com/package/@k6-contrib/fields-s3-images

it is using old ui for the upload with ref and all, did not have time to create one with reworked ui like in latest keystone.
